test/selenium/sessions_ui_test.js

```diff
@@ -255,6 +287,28 @@ describe('sessions ui', function() {
                            5000);
       });
     });
-  });
 
+    describe('tests changing views', function(){
+
+        before(function() {
+          driver.navigate().refresh();
+          setUpWebapp = new SetUpWebapp(driver);
+
+        });
+        beforeEach(function(){
+          return driver.wait(webdriver.until.titleIs('FoxBox'), 5000).then(
+            function() {
+          return setUpWebapp.getSignInPage();
+          }).then(function(signedInPageView) {
+             signedInPage = signedInPageView;
+          });
+        });
+
+        it('should go to sign out page' , function() {
+          return signedInPage.signOut().then(function(isSignedOut) {
+            assert.equal(isSignedOut, true);
```

Comment:
Nit: `isSignedOut` is actually a WebElement. Asserting on it to be true, is not totally correct. You can see it if you change this line by:
```js
assert.strictEqual(isSignedOut, true)
```

I would recommend is smaller (and also more implicit) like:
```js
return signedInPage.signOut(); // fails if we don't land on the signOut page
```